### PR TITLE
Add speed control with trigger

### PR DESCRIPTION
**Pull request by LucaSandoval**
I've added functionality to increase the speed while holding down the right trigger.
I've also set the default speed to 0.4, which seems to be the slowest the robot can go for some reason.

### CODE REVIEW

File: tests/pyfrc_test.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'''
```
Comment (zoecarver) [inline tests/pyfrc_test.py:1]: This should be another PR -- we'll also want some tests, not just the boilerplate.

**Review by zoecarver — CHANGES_REQUESTED** [review]
Please remove the cache file too

**Comment by julianallchin** [comment]
Was already added in an earlier push.
